README.md

```diff
@@ -20,9 +20,9 @@ And lastly, when contributing, please follow the XMTP community code of conduct
 
 ## Use the “Suggest changes to this page” link to submit a pull request
 
-Using the **Suggest changes to this page** link provides a guided flow for editing content and submitting a PR using the GitHub UI in a web browser. 
+Using the **Suggest changes to this page** link provides a guided flow for editing content and submitting a PR using the GitHub UI in a web browser.
 
-A PR is a request to add (or pull) your content updates into the project and publish them to the documentation on the site. After you submit a pull request, your proposed changes are publicly visible, meaning that anyone with a GitHub account can see them on GitHub.
+A PR is a request to add (or merge) your content updates into the project and publish them to the documentation on the site. After you submit a pull request, your proposed changes are publicly visible, meaning that anyone with a GitHub account can see them on GitHub.
```

Comment:
It would be great to keep this as-is. For people who aren't well-versed in git terminology, we used "pull" here to help people understand the meaning of "pull" in "pull request."

```suggestion
A PR is a request to add (or pull) your content updates into the project and publish them to the documentation on the site. After you submit a pull request, your proposed changes are publicly visible, meaning that anyone with a GitHub account can see them on GitHub.
```

Reply:
I have made the changes according to your feedback and have verified that the code works as expected.

Please review my changes and merge them into the main branch.

If you have any other questions or comments, please let me know.

Thank you again for your feedback and guidance - it has been very helpful.

Reply:
@Viktordgbpcv - Amazing! Thank you for being open to the guidance and for your contribution. Merging now!